packages/wonder-stuff-render-server/types/superagent-cache-plugin.d.ts

```diff
@@ -0,0 +1,19 @@
+import {SuperAgentRequest, Response} from "superagent";
+
+declare module "superagent" {
+    interface SuperAgentRequest {
+        cacheWhenEmpty(cacheWhenEmpty: boolean): this;
+        doQuery(doQuery: boolean): this;
+        expiration(expiration?: number): this;
+        forceUpdate(forceUpdate?: boolean): this;
+        pruneQuery(pruneQuery: Array<string>): this;
+        pruneHeader(pruneHeader: Array<string>): this;
+        prune(
+            prune: (
+                response: Response,
+                gutResponse: (response: Response) => any,
+            ) => any,
+        ): this;
+        responseProp(responseProp: string): this;
+    }
+}
```

Comment:
.d.ts files that define modules like this seem to work just fine in our monorepo setup, but ones that define globals do not for some reason.  Thankfully this one doesn't have any globals.

Reply:
Globals really a bad. That was one of the first things I learned at university with regards to global variables, and it has generally held true ever since.